### PR TITLE
* Fixing broken tests

### PR DESCRIPTION
**Pull request by maglub**
TL;DR:

* Removing checksum tests of Ubuntu 1804 choco package for the Windows tests, as the wsl choco package for ubuntu 1804 is broken, pending pull request https://github.com/bcurran3/ChocolateyPackages/pull/238
* Minor edit of README.md, to use become: true
* The suggested changes will fix the issues we have in Travis CI at the moment
* Fixing some typos in the yaml files

Longer story:

This pull request will not do much, other than take a step forward to fix the broken ci/cd tests, and a minor change to the README docu. The use of `remote_user: root` is not applicable to most environments, where `root` is not allowed to ssh into a system.

Note the changed `gitlab_runner_system_mode: yes` in the `tests/vars/defaults.yml` file, which was breaking the tests, as it broke quite a few things when the `ansible_user/remote_user <> root`.

The removal of the Windows test might be necessary, as the `choco install wsl-ubuntu-1804` breaks with a bad checksum. This is an open issue in https://chocolatey.org/packages/wsl-ubuntu-1804 since 6 months, not sure if it will ever be fixed there. The `choco install wsl-ubuntu-2004` breaks for other reasons. I have made a pull request for the choco package. https://github.com/bcurran3/ChocolateyPackages/pull/238

### CODE REVIEW

**Comment by riemers** [comment]
Thanks, its all green again!
